### PR TITLE
fix(advancedPane-event):高级面板添加自定义事件确定进行表单校验

### PR DESCRIPTION
**Pull request by ianxinnew**
# PR

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our [Commit Message Guidelines](https://github.com/opentiny/tiny-engine/blob/develop/CONTRIBUTING.md)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?
高级面板添加自定义事件确定按钮未进行表单校验，当输入为空时有“请输入正确浏览器事件”的提示弹出窗

Issue Number: N/A

## What is the new behavior?
根据设计统一，高级面板添加自定义事件确定按钮进行表单校验，当输入为空时去除“请输入正确浏览器事件”的提示弹出窗
## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: packages/settings/events/src/components/BindEvents.vue [hunk 1]
```diff
@@ -260,34 +260,38 @@ export default {
...
-              zh_CN: `${state.formData.methodDescription}的回调函数`
-            },
-            type: 'event',
-            functionInfo: {
-              params: [],
-              returns: {}
-            },
-            defaultValue: ''
+      ruleForm.value.validate((valid) => {
+        if (valid) {
```
Comment (chilingling) [inline packages/settings/events/src/components/BindEvents.vue]: 使用 early return，这样在 if 代码块里面就不用包裹一大串代码了
```javascript
if (!valid) {
  return
}
// ... other logic
```

File: packages/settings/events/src/components/BindEvents.vue [hunk 2]
```diff
@@ -260,34 +260,39 @@ export default {
...
+      ruleForm.value.validate((valid) => {
+        if (!valid) {
+          return
+        }
+        if (state.componentEvents[state.formData.methodName]) {
+          useNotify({
+            type: 'error',
+            message: `${state.formData.methodName}事件函数名已存在`
+          })
+          return false
```
Comment (ghost) [inline packages/settings/events/src/components/BindEvents.vue:284]: 同个函数中return返回的数据类型不一致

File: packages/settings/events/src/components/BindEvents.vue [hunk 3]
```diff
@@ -260,34 +260,39 @@ export default {
...
+                returns: {}
+              },
+              defaultValue: ''
+            }
+          })
+          state.showEventAdditive = !state.showEventAdditive
+        }
 
-      return undefined
+        return undefined
```
Comment (ghost) [inline packages/settings/events/src/components/BindEvents.vue]: 不必要的return，到这里已执行完，该函数绑定在了组件上，不需要返回值

**Comment by FairyShuai** [comment]
请问可以实现自定义校验规则吗

**Pull request closed without merging**